data_migrator.py

```diff
@@ -0,0 +1,104 @@
+from utils import get_connection, get_config
+import petl as etl
+
+def get_connection_to_dbs(source_config):
+    '''
+    Get the connection to the source and destination databases
+    :param: configs to connect to source DB
+    '''
+    from_db_conn = get_connection(source_config['host'], source_config['dbname'],
+                                  source_config['user'], source_config['pass'])
+    config = get_config()
+    to_db_conn = get_connection(config['host'], config['dbname'], config['user'], config['pass'])
+    return from_db_conn, to_db_conn
+
+def transfer_data(from_db_conn, to_db_conn):
+    '''
+    Transfer data from databases given cursor to execute queries to connected databases
+    Limitations:
+    1. poc.address_id is currently marked as  -1 since it was not provided in test data and is a FK non-null constraint
+    2. institution2poc table is not available in old schema
+    3. role table is already populated in bill.sql file so that table is skipped by this script
+    4. poc_poc_id is currently set to be poc_id since no relevant information is available about the column
+    5. project2moc_project.role_id column is not available in old schema and is a not null field in new schema
+        so we default it to 1 for now.
+    6. project2moc_project.username is not available from old schema so currently set to empty
```

Comment:
I split the project2poc table into project2poc and project2moc_poc.  Different information has to go into the tables.